CommonTools/RecoAlgos/plugins/PrimaryVertexSorter.h

```diff
@@ -98,11 +99,13 @@ PrimaryVertexSorter<ParticlesCollection>::PrimaryVertexSorter(const edm::Paramet
   if(produceOriginalMapping_){
       produces< CandToVertex> ("original");
       produces< CandToVertexQuality> ("original");
+      produces< VertexScore> ("original");
   }
   if(produceSortedVertices_){
       produces< reco::VertexCollection> ();
       produces< CandToVertex> ();
       produces< CandToVertexQuality> ();
+      produces< VertexScore> ();
```

Comment:
@slava77 this is the additional product in RECO/AOD with current RECO/AOD config